.github/dependabot.yml

```diff
@@ -5,8 +5,10 @@ updates:
   target-branch: master
   schedule:
     interval: daily
+  versioning-strategy: increase-if-necessary
```

Comment:
Let me know what your thoughts are regarding using `increase` here instead.

Reply:
@ripecosta I'm not the one who set this up in the first place and I'm not familiar with the options, can you make a case as to why one is better than the other?

Reply:
Sure thing. I'll also break the PR up into 3 separate ones so it's easier to review/merge.

Reply:
Updated the PR. All it does now is update dependencies. The other bits I put into:
- #1053 
- #1054 
- #1055